src/schema/types/models/User.ts

```diff
@@ -1,54 +1,69 @@
-import { PrismaClient } from "@prisma/client";
-import { objectType } from "nexus";
+import { interfaceType, objectType } from "nexus";
```

Comment:
```suggestion
import { objectType } from "nexus";
```

사용하지 않는 것은 import를 해제 하는 것이 좋다고 생각됩니다